### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by mmattbtw** [mmattDonk/AI-TTS-Donations#15] [Releases]
The bot is stable!
If you would like help setting up the bot, dm me on Discord/Twitter @mmattbtw/mmatt#9926
## What's Changed
* docs(README): Added documentation on how to run the bot by @MrAuro in https://github.com/mmattDonk/AI-TTS-Donations/pull/2
* feat(core): Release 1.0 is ready by @mmattbtw in https://github.com/mmattDonk/AI-TTS-Donations/pull/4

## New Contributors
* @MrAuro made their first contribution in https://github.com/mmattDonk/AI-TTS-Donations/pull/2
* @mmattbtw made their first contribution in https://github.com/mmattDonk/AI-TTS-Donations/pull/4

**Full Changelog**: https://github.com/mmattDonk/AI-TTS-Donations/commits/v1.0.0
**Changelog**: https://github.com/mmattDonk/AI-TTS-Donations/blob/main/CHANGELOG.md

<hr /><em>This discussion was created from the release <a href='https://github.com/mmattDonk/AI-TTS-Donations/releases/tag/v1.0.0'>v1.0.0</a>.</em>

**Comment by anqueue** [comment]
🎉
